full-stack-tests/core/src/frontend/standalone/BackgroundMap.test.ts

```diff
@@ -134,7 +134,7 @@ describe("Background map", () => {
         }
 
         expect(actualPixel.type).to.equal(Pixel.GeometryType.Surface);
-        expect(actualPixel.featureTable).not.to.be.undefined;
+        //  With the unified reality/terrain mesh, a feature table is present, so this test removed.... expect(actualPixel.featureTable).not.to.be.undefined;

```

Comment:
The line you commented out was testing that the feature table **was** defined. Your comment claims that a feature table is present, so why would it need to be commented out? If the behavior has changed, please update the test to verify the new behavior.

Reply:
Yes... removing that was not necessary.  Restored.